tyk-headless/templates/secrets.yaml

```
@@ -0,0 +1,16 @@
+{{- if .Values.SecretName.enabled -}}
```

To align with the other comments:
```suggestion
{{- if not .Values.secrets.useSecretName -}}
```